build.sbt

```diff
@@ -30,7 +30,13 @@ lazy val `cms-common` = project
 
 lazy val `cms-contentful` = project
   .configs(IntegrationTest)
-  .settings(commonSettings ++ commonTestSettings : _*)
+  .settings(commonSettings ++ commonTestSettings: _*)
+  .settings(
+    libraryDependencies ++= Seq(
+      "com.squareup.okhttp3" % "okhttp" % "3.2.0",
```

Comment:
here would be a comment appropriate that okhttp is used by contentful as optional dependency as HTTP client, otherwise it looks like an unused dependency and gets removed eventually